### DISCUSSION TITLE
关于waf的问题

### DISCUSSION THREAD

**Discussion by szh811218** [alibaba/higress#905] [General]
文档中提到安装waf插件，“在K8s集群中apply如下资源，以部署WAF插件”
但是在higress的控制台中，已经有waf插件的配置，我没明白到底要不要再安装waf的容器。

**Comment by CH3CHO** [comment]
能否贴一下问题中提到的文档链接？

**Reply by szh811218** [reply]
https://higress.io/zh-cn/docs/user/waf

**Reply by CH3CHO** [reply]
明白了。这个文档写的比较早。当时控制台还不支持插件配置，所以才需要手写YAML。现在直接在控制台上配置就行了，底层就是生成这段YAML。

**Reply by szh811218** [reply]
多谢了

**Comment by haimaren** [comment]
我使用的higress版本为1.40,开启waf插件后,dolphinscheduler(版本3.1.4)就打不开了(启动请求链没有正常启动完整),,,页面一片空白(问题同https://github.com/alibaba/higress/issues/808),,,关闭就立刻恢复正常,,,我试着调整connectionBufferLimits参数,重启higress-gateway,higress-controller pod也是同样的问题
![image](https://github.com/alibaba/higress/assets/26118792/8b921240-196d-421e-9660-a859ed5a820a)
这块域名都没有匹配到dolphinscheduler服务访问的域名,,,dolphinscheduler服务也会受到影响
![image](https://github.com/alibaba/higress/assets/26118792/383884fb-f304-486f-9e3e-2cae1e9350a0)

**Reply by CH3CHO** [reply]
你的用法不对。如果要配match domain，那么是在域名下面进行配置。全局配置都是全局生效。

**Reply by haimaren** [reply]
知道了,,,谢谢

**Reply by haimaren** [reply]
我对某个路由进行单独的waf配置(仅观测，不拦截),,,但是极个别页面还是出现了数据无法加载的问题,,,比如某个接口的参数"all=true",,,这块就加载不了,,,使用工具单独调用接口数据返回正常,,,关闭waf访问页面也没有问题,,,感觉和之前dolphinscheduler页面打不开很类似,,,不清楚是什么原因造成的

**Reply by CH3CHO** [reply]
这个“极个别页面”是开了WAF的路由下面的页面吗？

**Reply by haimaren** [reply]
开了waf的路由是后端接口,,,前端页面是独立出来的
